### ISSUE TITLE
Plugin stops working if another program which supports discord rich presences

### ISSUE THREAD

**Issue by kakuttaja** [NaamloosDT/foo_discord#8]
Plugin stops working the instant a program that supports discord rich presences is active. Removing the game from the "Games" tab in settings doesn't help the problem, but of course discord doesn't display the name of the game anymore on your profile.

The plugin just stops doing its' job and discord now only shows the "Playing foobar2000" but no more extra info like the plugin does.

Program I tested it with is RuneLite with its discord plugins all disabled, and game removed from discord.

**Comment by Naamloos** [comment]
Play a new song

**Comment by kakuttaja** [comment]
Tried that. Also let foobar2000 to automatically start a new song instead of manually selecting one, but problem still remained.

**Comment by Naamloos** [comment]
Really should not be possible, are you sire you didn't disable the plugin in foobar settings?

**Comment by kakuttaja** [comment]
I've tried to reproduce the problem, but I've been unable to do so now. After a few hours I went and opened up RuneLite again, and the problem did not occur. I've tried restarting both RuneLite, Foobar2000 and Discord to see if it might come back, but it wont. So the problem was magically(?) solved.

Originally when I had the problem, RuneLite had overwritten Foobar2000 on the "Currently playing", which might've lead to the problem? Now I don't have any issues, and Foobar2000 is always the "Currently playing" even when RuneLite is added to the "Games" -list in discord settings and is active. This might be related, but I myself have no idea. Just my observations.

Problem solved, I guess?
